tests/annotation_model_tests/settings.py

```diff
@@ -0,0 +1,3 @@
+model_path = "C:/repos/satellite-images-nso-datascience/saved_models/Superview_Nieuwkoopse_plassen_20190302_113613_to_20221012_104900_random_forest_classifier.sav"
```

Comment:
Can we make this a relative path and include it in the repo? Otherwise it becomes impossible to run this test out of the box.

Reply:
It's not longer an absolute path, but the .sav files are not in the repository (see also my comment on test_settings.py), so someone else still wouldn't know how to run these tests.

Reply:
you should download from the link in the saved_models file but automatic downloading from the bobstorage would not hurt

Reply:
also the models is very large like 250mb, for now out of this scope to make a smaller model

Reply:
fixed by using smaller models